src/routes/submit/submit-validation.js

```diff
@@ -0,0 +1,56 @@
+import { getQuestionFromSections } from '../../common/helpers/data-extract/data-extract.js'
+
+/**
+ * @import {Request} from "@hapi/hapi/lib/types/request.js"
+ * @import {SectionData, QuestionAnswerData} from "../../common/helpers/data-extract/data-extract.js"
+ */
+
+/**
+ * @param {Request} request
+ * @returns {boolean}
+ */
+export const isValidRequest = (request) => {
+  const headers = request.headers
+  const contentType = headers['content-type'] || headers['Content-Type']
+  const isValidContentType =
+    contentType && contentType.includes('application/json')
+  if (!isValidContentType) {
+    request.logger.warn(
+      'Invalid request. Content-Type header is missing or it does not include application/json'
+    )
+  }
+  return !!isValidContentType
+}
+
+/**
+ * @param {Request} request
+ * @returns {boolean}
+ */
```

Comment:
I'm not sure this signature definition is applied to any given function :thinking: ?

Reply:
removed